.github/container/Dockerfile.base

```diff
@@ -12,11 +12,16 @@ ARG GIT_USER_NAME
 ARG SRC_MANIFEST_FILE
 ARG DEST_MANIFEST_DIR
 ARG CLANG_VERSION
+ARG NVIDIA_REQUIRE_CUDA
 
 ###############################################################################
 ## Install Python and essential tools
 ###############################################################################
 
+# Temporary WAR to run container on arm64 with CUDA-12.3
+RUN test -n "${NVIDIA_REQUIRE_CUDA}"
```

Comment:
This would break amd64 builds if NVIDIA_REQUIRE_CUDA is not provided, even though the variable is already encoded in the base image, e.g. the amd64 version of nvidia/cuda:12.x.

Reply:
It'll cause problem if we merge this into the 24.04-devel branch, which  is used for the NGC release.